etest/qemu.py

```diff
@@ -0,0 +1,48 @@
+"""QEMU interpreter management."""
+
+# Please see https://github.com/multiarch/qemu-user-static
+
+import logging
+from typing import Any
+
+from etest import docker
+
+
+class qemu:
+    """QEMU interpreter management."""
+
+    def __init__(self, logger: logging.Logger, arch: str) -> None:
+        """Initialize."""
+        self.logger = logger
```

Comment:
The logger should be managed by the module and **not** passed to the object at initialisation.